runtime/local/src/lib.rs

```diff
@@ -1448,6 +1448,70 @@ impl_runtime_apis! {
         }
     }
 
+    impl moonbeam_rpc_primitives_debug::DebugRuntimeApi<Block> for Runtime {
```

Comment:
Looks like this impl is mostly copypaste?  Maybe we should extract it into a single module, used by all runtimes?

Reply:
That’s right, usually multi-runtimes projects create something like `common`  module with shared implementations. Problem here it’s under macros and move it outside makes code more and more sophisticated.